android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -480,13 +539,16 @@ public void testNonASCII() throws CouchbaseLiteException {
     //region Helper methods
     private void testWithConfiguration(LogLevel level, LogFileConfiguration config, Runnable r) {
         LogFileConfiguration old = Database.log.getFile().getConfig();
+        EnumSet<LogDomain> domains = Database.log.getConsole().getDomains();
         Database.log.getFile().setConfig(config);
         Database.log.getFile().setLevel(level);
         try {
             r.run();
         } finally {
             Database.log.getFile().setLevel(LogLevel.INFO);
             Database.log.getFile().setConfig(old);
+            Database.log.getConsole().setDomains(domains);
+            Database.log.getConsole().setLevel(LogLevel.INFO);
```

Comment:
The default console log level is WARNING. Should we reset it to WARNING?

Reply:
In iOS, we are using the same as file logging backup level. I can change this to warning.